BCStudentSoftwareDevTeam/celts
# Datepicker volunteer profile admin view issue#5

**Issue:** 
The datepicker for the ban modal allows me to pick a date that is before the current date and the current date. It shouldn't allow for either.

**Solution:** 
Added a new date picker in the ban modal that only allows admins to set the end ban date to a date after today. I used some Jquery datepicker methods and changed the type of the datepicker to date.

**To Test:** 

1. Navigate to the IP Address/profile/ramsayb2 
2. Click on "Edit"
3. Click on the datepicker input field
4. You would see the date picker widget, which only allows you to select an end the date later than today.

![image](https://user-images.githubusercontent.com/57919439/147981633-01ed33f2-3b17-4bd0-9720-458ee37a5114.png)
